tests/unit/actions/clickable-test.js

```diff
@@ -1,62 +1,70 @@
 import { test } from 'qunit';
 import { moduleFor, buildProperty } from '../test-helper';
-import clickable from '../../page-object/properties/clickable';
+import { create, clickable } from '../../page-object';
 
-moduleFor('Actions', 'clickable');
+moduleFor('.clickable');
 
 test('calls Ember\'s click helper', function(assert) {
   assert.expect(1);
 
-  let expectedSelector = 'button',
-      property;
+  let page;
 
   window.click = function(actualSelector) {
-    assert.equal(actualSelector, expectedSelector);
+    assert.equal(actualSelector, 'span');
   };
 
-  property = buildProperty(clickable(expectedSelector));
+  page = create({
+    foo: clickable('span')
+  });
 
-  property.invoke();
+  page.foo;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/san650/ember-cli-page-object/file/1817059231/issues/source?bid=2903707&fileBranchId=3076256#l20)